inc/saivlan.h

```diff
@@ -119,6 +132,14 @@ typedef enum _sai_vlan_attr_t
      */
     SAI_VLAN_ATTR_VLAN_ID = SAI_VLAN_ATTR_START,
 
+    /**
+     * @brief Vlan Type
+     *
+     * @type sai_vlan_type_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
```

Comment:
making this mandatory on create breaks backward compatibility with creating vlans,
would it be possible to make this create_only and assign default value?

Reply:
Agree. This attribute can be CREATE_ONLY optional param with default value, "SAI_VLAN_TYPE_DATA". I will push the changes.